.github/ISSUE_TEMPLATE/docs.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Docs
+about: Found a typo in docs? You can use this one!
```

Comment:
Maybe we could also add- "for documentation requests/ideas/issues?"

Reply:
I am not sure I quite understand. Can you clarify please, @Vyvy-vi?

Reply:
oh wupps

Reply:
I read the filename wrong 😓